week3/homework/src/index.js

```diff
@@ -1,3 +1,100 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const express = require('express');
+const bodyParser = require('body-parser');
+const fs = require('fs');
+
+const app = express();
+
+app.use(bodyParser.json());
+
+// addTodo(POST / todos /)
+//adds todo item
+app.post('/todos', (request, response) => {
+  fs.closeSync(fs.openSync('./todos.txt', 'a'));
+  fs.readFile('./todos.txt', 'utf8', (error, todoList) => {
+    if (error) {
+      console.error(error);
+    } else {
+      const todos = todoList.trim() == '' ? [] : JSON.parse(todoList);
+      todos.push({
+        text: request.body.todo,
+        done: '',
+      });
+      fs.writeFile('./todos.txt', JSON.stringify(todos), error => {
+        console.error(error);
+        response.json({ result: request.body.todo });
```

Comment:
You need to check the error and send the response code accordingly

Reply:
Done!